pumping/src/main/kotlin/com/dpm/pumping/auth/config/CurrentUserResolver.kt

```diff
@@ -0,0 +1,31 @@
+package com.dpm.pumping.auth.config
+
+import com.dpm.pumping.user.domain.UserRepository
+import jakarta.servlet.http.HttpServletRequest
+import org.springframework.core.MethodParameter
+import org.springframework.stereotype.Component
+import org.springframework.web.bind.support.WebDataBinderFactory
+import org.springframework.web.context.request.NativeWebRequest
+import org.springframework.web.method.support.HandlerMethodArgumentResolver
+import org.springframework.web.method.support.ModelAndViewContainer
+
+@Component
+class CurrentUserResolver(
+    private val userRepository: UserRepository
+) : HandlerMethodArgumentResolver {
```

Comment:
헤헤 한 줄 빈칸 띄워주세용

Reply:
`{`요거 말하는거 맞나여?
```suggestion
) : HandlerMethodArgumentResolver
{
```
이런 느낌?